app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailActivityPresenter.java

```diff
@@ -71,12 +71,12 @@ private void processEventAndDisplay(Event event) {
         List<Ticket> tickets = event.getTickets();
 
         if(tickets != null) {
-            quantity = 0;
+            totalTickets = 0;
```

Comment:
initialize `totalTickets` outside the case to avoid exception when `tickets == null`